GooglePlayInstant/Samples/TestApp/Scripts/TestApp.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using UnityEngine;
+
+namespace GooglePlayInstant.Samples.TestApp
+{
+    /// <summary>
+    /// Tests instant app plugin features through button clicks
+    /// </summary>
+    public class TestApp : MonoBehaviour
+    {
+        private const string CookieEmoticon = "\uD83C\uDF6A";
+        private string _storedCookie;
+
+        /// <summary>
+        /// Sets the instant app cookie to a unique string
+        /// </summary>
+        public void BtnEvt_WriteCookie()

```

Comment:
We generally avoid abbreviations and underscores in non-test methods. So maybe drop the "BtnEvt_" prefix"

Reply:
That's fair, I wanted to indicate that these functions were referenced by a button in the scene. Just in case someones trying to figure out where they're called from and their IDE is telling them they aren't referenced in script.